R/utils.R

```diff
@@ -249,3 +249,23 @@ add_report <- function(data, report, name = NULL) {
 
   data
 }
+
+
+#' Add an element to the report object
+#'
+#' @param x an object of class `data frame` or `linelist`
+#' @param name the name of the cleaning operation
+#' @param add_this the object to add to the report object
+#'
+#' @return the input report object with an additional element
+#' @keywords internal
+#'
+add_to_report <- function(x, name, add_this = NULL) {
```

Comment:
Suggestion.Since this function is essentially populating a key/value store consider renaming your parameters to reflect it. e.g. `function(x, key value = NULL)`